app/1.0/blog/2016-08-18-polymer-starter-kit-or-polymer-cli

```diff
@@ -0,0 +1,49 @@
+---
+title:  "Should I Use Polymer Starter Kit or Polymer CLI"
+---
+
+For many developers new to Polymer, the [Polymer Starter Kit](https://github.com/polymerelements/polymer-starter-kit) project is a useful seed to get the ball rolling on their own app. With the release of [Polymer CLI](https://www.polymer-project.org/1.0/docs/tools/polymer-cli), which contains its own starter projects, it became unclear to developers which resource they should choose when creating a new project.
```

Comment:
`https://github.com/polymerelements/polymer-starter-kit`
to
`https://github.com/PolymerElements/polymer-starter-kit`